docker-compose.xdebug.override.yml.dist

```diff
@@ -0,0 +1,31 @@
+version: "3.7"
+
+services:
+  # @NOTE: enable ssh_auth_sock environment and volume variables
+  # in the case you need to access to remote servers
+  # inside docker using self-hosted credentials. It only works on Linux.
+#  php:
```

Comment:
These lines are already present at docker-compose.override.yml.dist . If the debugging will be managed by its own docker-compose file, then the lines from docker-compose.override.yml.dist must be removed.

Reply:
The only lines I would preserve in docker-compose.override.yml.dist would be those who belong to the SSH_AUTH_SOCK configuration.